### ISSUE TITLE
Error using pre-trained model with CARD model

### ISSUE THREAD

**Issue by saifhassan** [vturrisi/solo-learn#341]
Hello @DonkeyShot21 @vturrisi,

I want to train [CARD Model](https://github.com/XzwHan/CARD/tree/main) (Classification Part) and use pre-trained model (generated by `main_pretrain.py`) as f_phi pretrained model so that it can improve accuracy.

What configuration settings, I will have to do so that I can use with CARD?

Thanks

**Comment by vturrisi** [comment]
I'm not familiar with the repo that you linked. After you pretrain a model with solo, you can use the weights saved in the checkpoint and load it accordingly. You will need to check how to integrate it afterwards.
